Parsers/ASimAuthentication/Parsers/ASimAuthenticationAADNonInteractive.yaml

```diff
@@ -15,23 +15,27 @@ References:
 Description: |
     This ASIM parser supports normalizing Azure Active Directory Non Interactive sign in logs, stored in the AADNonInteractiveUserSignInLogs table, to the ASIM Authentication schema.
 ParserName: ASimAuthenticationAADNonInteractiveUserSignInLogs
+EquivalentBuiltInParser: _ASim_Authentication_AADNonInteractiveUserSignInLogs
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Sorted the columns